src/utils/utils.js

```diff
@@ -35,8 +35,8 @@ export const getConfigObjectsForAnalytics = (configurations, groupCode) => {
     const numeratorsInGroup = configurations.numerators.filter((numerator) => {
         return (
             members.includes(numerator.code) &&
-            numerator.dataID !== null &&
-            numerator.dataSetID !== null
+            numerator.dataID &&
```

Comment:
these updates are because when testing I saw in the configuration values on demos.dhis2.org/dq, there were some numerators where dataID was undefined (and not expressly null)